frontend/lib/screens/auth/login/login_screen.dart

```diff
@@ -40,45 +51,294 @@ class LoginScreen extends StatelessWidget {
         width: double.infinity,
         height: double.infinity,
         child: SingleChildScrollView(
-          child: Column(
-            crossAxisAlignment: CrossAxisAlignment.center,
-            mainAxisAlignment: MainAxisAlignment.start,
-            children: [
-              const SizedBox(
-                height: 45,
-              ),
-              CustomButtonComponent(
-                text: 'Log in with Google',
-                width: 328,
-                height: 48,
-                iconPath: 'assets/icons/google_icon.png',
-                onPressed: () {
-                  // Handle button press
-                },
-              ),
-              const SizedBox(
-                height: 45,
-              ),
-              Container(
-                padding: const EdgeInsets.symmetric(
-                  horizontal: 40,
-                ),
-                child: const DividerWithTextComponent(
-                  text: 'or log in with an email',
-                  textSize: 24,
-                ),
-              ),
-              const SizedBox(
-                height: 45,
-              ),
-              LoginFormWidget(),
-              const SizedBox(
-                height: 45,
-              ),
-            ],
+          child: BlocConsumer<LoginBloc, LoginState>(
+            listener: (context, state) {
+              _blocListener(
+                  state, emailTextController, passwordTextController, context);
+            },
+            builder: (context, state) {
+              return _blocBuilder(loginBloc, state, emailTextController,
+                  passwordTextController, formKey, context);
+            },
           ),
         ),
       ),
     );
   }
 }
+
+void _blocListener(
+  LoginState state,
+  TextEditingController emailTextController,
+  TextEditingController passwordTextController,
+  BuildContext context,
+) {
+  if (state is LoginEdited) {
+    emailTextController.text = state.email;
+    passwordTextController.text = state.password;
+  }
```

Comment:
We could remove this, as we discussed it is not needed

Reply:
Yes, I will do this in the PR #23